app/javascript/containers/OptionsContainer.jsx

```diff
@@ -8,6 +8,7 @@ import FreeOption from "../components/FreeOption";
 import Status from "../components/Status";
 import DependentQuestions from "../components/DependentQuestions";
 
+import { getEmail } from "../selectors/usersSelector"
```

Comment:
If I remember correctly, we decided not to track the editor of a question anymore and just go by login times instead - if this isn't being used anymore, can we get rid of all instances of `getEmail`, `editor_email`, etc. in the code?